cmd/copy.go

```diff
@@ -638,6 +640,22 @@ func (raw rawCopyCmdArgs) cook() (CookedCopyCmdArgs, error) {
 		cooked.preserveSMBInfo = false
 	}
 
+	if raw.preservePOSIXProperties == "" && areBothLocationsPOSIXAware(cooked.FromTo) {
+		if cooked.FromTo.IsUpload() {
+			raw.preservePOSIXProperties = common.EPosixPropertiesOption.FullFidelity().String()
+		} else if cooked.FromTo.IsDownload() {
+			raw.preservePOSIXProperties = common.EPosixPropertiesOption.SpecialFilesAndProperties().String() // todo: should we default to FullFidelity? It may not always work out well for the customer.
+		} // ignore BlobBlob, these properties persist via metadata & it doesn't make sense to explicitly ignore these bits of metadata.
```

Comment:
What is "blobblob" ?

Reply:
BlobBlob is a service-to-service transfer between two Azure Blob Storage endpoints.